lambda-functions/Lambda/psycopg2-no-SQLAlchemy/lambda-reddit-streaming.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import praw
+from access_psycopg2 import make_conn,fetch
+
+# thanks Mac for the Reddit credentials
+
+REDDIT_CLIENT_ID = "VXYCrQAymaBrLQ"
+REDDIT_CLIENT_SECRET = "0Wwvv9zptchx7iqweK52cvbtX4A"
```

Comment:
Make sure to remove and refresh these credentials

Reply:
Gotcha